packages/cli/src/readCliVersionFromPackageJson.ts

```diff
@@ -15,10 +15,14 @@
  */
 
 import * as fs from "node:fs";
+import { dirname } from "node:path";
+import { fileURLToPath } from "node:url";
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
```

Comment:
uber uber nit: why keep a global variable you need once?

Reply:
Because it was a comfy place to copypaste obviously.